packages/parse5/package.json

```diff
@@ -33,8 +33,12 @@
     "types": "dist/index.d.ts",
     "exports": {
         ".": {
-            "import": "dist/index.js",
-            "require": "dist/cjs/index.js"
+            "import": "./dist/index.js",
+            "require": "./dist/cjs/index.js"
+        },
+        "./dist/*": {
+            "import": "./dist/*",
+            "require": "./dist/cjs/*"
```

Comment:
I think that this is an anti-pattern.
The export map allows for an “allow-list” of what APIs to export as separate entry points, that is *unrelated* to the file system.
I believe we should carefully choose to expose what we want folks to use instead of exposing the whole file system?

Perhaps the problem in that script can be solved in a different way?